src/internal/api/packages/find.go

```diff
@@ -6,18 +6,18 @@ package packages
 
 import (
 	"fmt"
+	"io/fs"
 	"net/http"
 	"os"
+	"path/filepath"
 	"regexp"
 
 	"github.com/defenseunicorns/zarf/src/internal/api/common"
 	"github.com/defenseunicorns/zarf/src/pkg/message"
-	"github.com/defenseunicorns/zarf/src/pkg/packager"
-	"github.com/defenseunicorns/zarf/src/pkg/utils"
 )
 
-var packagePattern = regexp.MustCompile(`zarf-package-.*\.tar`)
-var initPattern = regexp.MustCompile(packager.GetInitPackageName(""))
+var packagePattern = regexp.MustCompile(`zarf-package.*.tar\.zst$`)
```

Comment:
```suggestion
var packagePattern = regexp.MustCompile(`zarf-package-.*\.tar`)
```

Packages can be created uncompressed, (ie `zarf-package-name.tar`), so the `.zst` extension cannot be used an an identifier.

Reply:
I think there might be bigger issues that will have to be looked into because the zarf cache gives this error
Unable to read the package at: `%2FUsers%2Fmikewinberry%2F.zarf-cache%2Fzarf-init-amd64-v0.25.0-20-g75b142a4.tar.zst` Its the only package that seems to be unreadable.